### ISSUE TITLE
[Feature Request] add CLI flags to hide/show/enter directory paths

### ISSUE THREAD

**Issue by faelin** [Canop/broot#558]
_[Pulled from comments on Issue #409]_

It would be extremely useful to be able to pass special-paths to broot via CLI flags, as in
```bash
broot --hide="/example/useless" --no-enter="/example/slow-remote-mount" --no-enter="**/.git" --enter="/example/git-ignored-path"
```

In addition, per Issue #551, this feature should be coupled with new Verb commands within the Broot view itself:

| `:hide`<br>(with selection)  | `:show-hidden`<br>(with argument) | `:no-enter`<br>(with argument) |
| --- | --- | --- |
| ![Screen Shot 2022-06-07 at 11 49 03 AM](https://user-images.githubusercontent.com/8279240/172424921-919644a6-a222-4738-8c3c-42717e1225d8.png)  |  ![Screen Shot 2022-06-07 at 11 49 19 AM](https://user-images.githubusercontent.com/8279240/172424925-5c6364f0-b211-4cc2-866d-02f6016653a8.png) | ![Screen Shot 2022-06-07 at 11 49 28 AM](https://user-images.githubusercontent.com/8279240/172424928-bdcf547f-0d8d-4308-8b0b-c5ffb254f185.png) |

**Comment by Canop** [comment]
OK.

BTW I should probably have a mode which doesn't render the UI when you only want a `:pt` or a `:pp`

**Comment by faelin** [comment]
Yes, that would be good. Another case where a command-line flag might be the best mechanism, as in `-q`, or perhaps `-p`/`--print`
